bindings/java/nz/mega/sdk/MegaApiSwing.java

```diff
@@ -2,8 +2,24 @@
 
 import javax.swing.SwingUtilities;
 
+/**
+ * Control a MEGA account or a shared folder using a Java Swing GUI.
+ */
 public class MegaApiSwing extends MegaApiJava {
 
+    /**
+     * Instantiates a new MEGA API using Swing.
+     *
+     * @param appKey
+     *              AppKey of your application. You can generate your AppKey for free here: <br>
+     *              https://mega.co.nz/#sdk
+     * @param userAgent
+     *              User agent to use in network requests. If you pass null to this parameter,
```

Comment:
It's a good idea to *always* use full sentences, that is: end them with a full stop. It reads awkwardly to have two sentences, and the second one isn't terminated. Not a big issue, but it's good practice to terminate *all* sentences (even single ones) in the doc strings.